tensorflow/java/src/test/java/org/tensorflow/op/core/GeneratedOperationsTest.java

```diff
@@ -40,4 +41,20 @@ public void tensorInputTensorOutput() {
       }
     }
   }
+
+  @Test
+  public void testListInputTensorOutput() {
+    try (Graph g = new Graph();
+        Session sess = new Session(g)) {
+      Ops ops = Ops.create(g);
+      ArrayList<Operand<Integer>> inputs = new ArrayList<Operand<Integer>>();
+      inputs.add(ops.constant(1));
+      inputs.add(ops.constant(2));
+      inputs.add(ops.constant(3));
+      Operand<Integer> x = ops.math().addN(inputs);
```

Comment:
Just to make sure I understand, we didn't _need_ `ArrayList`, right? This could have been:

```java
Ops ops = Ops.create(g);
Operand<Integer> x = ops.math().addN(new Operand<Integer>{ops.constant(1), ops.constant(2), ops.constant(3)});
```

?

Reply:
We were talking about this a bit and at least when I tried that didn't work (assuming you mean for it to be an array of Operands). This is because the inputs that are marked as iterable in the call to TypeOf are generated to take an [Iterable](https://docs.oracle.com/javase/8/docs/api/java/lang/Iterable.html), so I get an error
```
required: Iterable<Operand<T>>
found: Operand<Integer>[]
```
My Java is definitely a little out of date and I might have misunderstood the question, so I could definitely be missing something. We can also update the generate code more if this isn't the intended interface.

CC @melissagrueter @irenedea

Reply:
Ah, righto. Okay, fine for now (this PR should be merged shortly, we can revisit this later if needed).